src/main/app/src/app/admin/parameter-edit/parameter-edit.component.ts

```diff
@@ -47,10 +51,13 @@ export class ParameterEditComponent extends AdminComponent implements OnInit {
     userEventListenerParameters: UserEventListenerParameter[] = [];
     zaakbeeindigParameters: ZaakbeeindigParameter[] = [];
     selection = new SelectionModel<ZaakbeeindigParameter>(true);
+    mailtemplateKoppelingen: MailtemplateKoppelingEnum[] =
```

Comment:
deze moet nu heten mailtemplateKoppelingEnums of als je die class hebt hernoemt mailtemplateKoppelingSoorten of mailtemplateKoppelingTypes of mailtemplateKoppelingFuncties

Reply:
Aangepast